rpc/server.go

```diff
@@ -0,0 +1,602 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// Copyright 2020 The Darwinia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
Modified the source code of `net/rpc` and `net/rpc/jsonrpc`, the LICENSE needs to be reviewed.

Reply:
BSD License should be good